src/shared/copy.c

```diff
@@ -896,6 +898,11 @@ static int fd_copy_directory(
         assert(st);
         assert(to);
 
+        if (set_contains(denylist, st)) {
```

Comment:
hmm, now you only apply this to directory inodes. I'd really apply it to any kind of inode.

i.e. it hink this check should be moved inside of the FOREACH_DIRENT_ALL() loop below, so that we just `continue` in it, to skip an entry.

Reply:
We also apply it in copy_leaf_node() so it should apply to everything